src/compute/multiply.ts

```diff
@@ -0,0 +1,49 @@
+import { Image } from '../Image';
+import { getOutputImage } from '../utils/getOutputImage';
+import { validateChannels } from '../utils/validators/validators';
+
+export interface MultiplyOptions {
+  /**
+   * Channels where value will be multiplied.
+   * @default all channels
+   */
+  channels?: number[];
+  /**
+   * Image to which the resulting image has to be put.
+   */
+  out?: Image;
+}
+
+/**
+ *
+ * Multiplies points by a certain value.
+ * @param image - image to whcih multiplication will be applied.
+ * @param value - Value by which each pixel will be multiplied.
+ * @param options - Multiply options
+ * @returns image.
+ */
+export function multiply(
+  image: Image,
+  value: number,
+  options: MultiplyOptions = {},
+) {
+  const {
+    channels = new Array(image.components).fill(0).map((value, index) => index),
+  } = options;
+  validateChannels(channels, image);
+
+  const newImage = getOutputImage(image, options, { clone: true });
```

Comment:
Why didn't you provide the `out` option which let's you choose where to put the result, in the same way we do in many other processing methods?

Reply:
I looked at several functions as examples(level, invert, pixelate) so I did it the same way. What did I do wrong?

Reply:
Sorry I missed something while reviewing